src/utils/navigatorIndex.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import { INDEX_ROOT_KEY } from 'docc-render/constants/sidebar';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
lets remove this line and add proper JSDoc like the rest.

Reply:
Resolved 19f92322e60a5702a0c09780cdc14da1b22327b4